CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased](https://github.com/rerun-io/rerun/compare/latest...HEAD)
 
+# ⚠️ Breaking changes
+* `mesh_material: Material` has been renamed to `albedo_factro: AlbedoFactor` [#6841](https://github.com/rerun-io/rerun/pull/6841)
```

Comment:
good call to add it right away! Wondering if we should also add the same note in the migration guide from the start? I know its redundant, but we already have it and it's a good place to dump the micro code snippet you already put on the PR description :)